### PR TITLE
Update prettier version and normalize all files

### PR DESCRIPTION
**Pull request by weshaggard**
This is step 1 of cleaning up how we run prettier in the specs repo. Once this PR goes in I will also update the unified pipeline to use the same version of prettier we are using here.

### CODE REVIEW

File: specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_deleted.json [hunk 1]
```diff
@@ -11,4 +11,4 @@
     "syncToken": "zAJw6V16=Njo1IzUxNjQ2NzM=;sn=5164673"
   },
   "specversion": "1.0"
-}
\ No newline at end of file
+}
```
Comment (mikeharder) [inline specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_deleted.json:14]: It looks like the generated swaggers were previously missing a trailing newline.  Is this fixed in the latest version of `@azure-tools/typespec-autorest`?

**Review by mikeharder — APPROVED** [review]

File: specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_deleted.json (hunk 1 above)
Comment (weshaggard) [reply specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_deleted.json:14]: These are examples and they were actually even flagged in the PR but that check was overridden. I don't think this is the typespec-autorest issue.

Comment (mikeharder) [reply specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_deleted.json:14]: I believe the examples are created by typespec-autorest though?  I can test this on the latest version of typespec-autorest to make sure the generated examples contain a trailing newline.

File: documentation/ci-fix.md [hunk 2]
```diff
@@ -10,18 +10,14 @@ If your problem is some existing error name that is not a word and need to supre
...
 npm install
-npm run prettier-fix
-```
-
-Or if you want to fix specified service, use the complete path, not relative:
-
-```
-npm install
-npm run prettier -- --write "<path to repo>/azure-rest-api-specs/specification/**/*.json"
+npx prettier -w **/*.json
```
Comment (raych1) [inline documentation/ci-fix.md:20]: new version of prettier installed here might bring validation gaps comparing to unified pipeline.

File: specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_deleted.json (hunk 1 above)
Comment (raych1) [reply specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_deleted.json:14]: Most of these examples are written manually.

File: specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_modified.json [hunk 3]
```diff
@@ -11,4 +11,4 @@
     "syncToken": "zAJw6V16=Njo1IzUxNjQ2NzM=;sn=5164673"
   },
   "specversion": "1.0"
-}
\ No newline at end of file
+}
```
Comment (konrad-jamrozik) [inline specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_modified.json:14]: Maybe move these new line additions to a separate PR to keep the diff clean?

File: specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_deleted.json (hunk 1 above)
Comment (konrad-jamrozik) [reply specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_deleted.json:14]: Maybe let's move these newline additions to a separate PR to keep the diffs clean?

Comment (weshaggard) [reply specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/examples/cloud-events-schema/keyvalue_deleted.json:14]: I included the newline diff here as well because I wanted the entire repo to be prettier clean as part of the upgrade.
